quipucords/api/scanjob/serializer.py

```diff
@@ -109,7 +111,7 @@ class ScanJobSerializerV1(NotEmptySerializer):
     status = ValidStringChoiceField(read_only=True, choices=ScanTask.STATUS_CHOICES)
     status_message = CharField(read_only=True)
     tasks = TaskField(many=True, read_only=True)
-    options = ScanOptionsSerializer(read_only=True, many=False)
+    options = DictField(required=False, default={})
```

Comment:
Using a literal `{}` might be problematic. This instance will be modified on the fly if default value is used.
Does `default` accepts a "factory" like `dict`?

Reply:
👍 yes, I see we do that in other defaults too.